Formula/cmusfm.rb

```diff
@@ -0,0 +1,67 @@
+class Cmusfm < Formula
+  desc "Last.fm standalone scrobbler for the cmus music player"
+  homepage "https://github.com/Arkq/cmusfm"
+  url "https://github.com/Arkq/cmusfm/archive/v0.3.3.tar.gz"
+  sha256 "9d9fa7df01c3dd7eecd72656e61494acc3b0111c07ddb18be0ad233110833b63"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libfaketime" => :test
+
+  def install
+    system "autoreconf", "--install"
```

Comment:
@puxlit Does the configure script not work out of the box? Do we always have to run `autoreconf`?

Reply:
As yet, the [source repo](https://github.com/Arkq/cmusfm/tree/v0.3.3), does _not_ include a pre-built configure script. Is this a problem, given Homebrew's focus on bottling everything such that the vast majority of users aren't going to be installing any build-time dependencies?

Reply:
In lieu of additional feedback, I'm marking this thread as resolved.